src/Wordsmith.php

```diff
@@ -100,6 +103,10 @@ function (Event $event) {
 			}
 		);
 
+
+		// Register GraphQL functionality
+        $this->registerGraphQL();
```

Comment:
I appreciate the intent of sequestering this into its own method, but `init()` is pretty short here; let's just leave the GQL event code directly in here with the other Event snippet.